runtime/lua/vim/treesitter.lua

```diff
@@ -44,50 +46,28 @@ function Parser:set_included_ranges(ranges)
   self.valid = false
 end
 
-local M = {
-  parse_query = vim._ts_parse_query,
-}
+-- TODO(vigoux): not that great way to do it, but that __index method bothers me...
+local M = vim.tbl_extend("error", query, lang)
 
 setmetatable(M, {
   __index = function (t, k)
       if k == "TSHighlighter" then
-        t[k] = require'vim.tshighlighter'
+        t[k] = require'vim.treesitter.highlighter'
```

Comment:
I don't think `TSHighlighter` was around in 0.4.3, so we can probably remove this entirely and just return this for `highlighter` instead.

Alternatively, we can issue a "please use `treesitter.highlighter` when accessing highlights` message for awhile to let people switch and then change it afterwards.